src/libraries/DrawAccumulatorLib.sol

```diff
@@ -34,6 +38,12 @@ library DrawAccumulatorLib {
         uint32 second;
     }
 
+    /// @notice Adds balance for the given draw id to the accumulator.
+    /// @param accumulator The accumulator to add to
+    /// @param _amount The amount of balance to add
+    /// @param _drawId The draw id to which to add balance to. This must be greater than or equal to the previous addition's draw id.
+    /// @param _alpha The alpha value to use for the exponential weighted average.
```

Comment:
So `_alpha` here is different from the `_smoothing` value in the PrizePool contract?